resources/brave-youtube-navigation-fix.js

```diff
@@ -0,0 +1,12 @@
+let previous = 0;
+
+(function (now) {
+  window.performance.now = function () {
+    let current = now.apply(this);
+    if (current <= previous) {
+      current = previous + Number.EPSILON * 100000;
+    }
+
+    return previous = current;
+  }
+})(window.performance.now);
```

Comment:
no need to pass this in as a parameter to the closure, since its a global structure.